public/js/gulliver.es6.js

```diff
@@ -60,8 +56,8 @@ class Gulliver {
     this.analytics.trackPageView(window.location.href);
   }
 
-  _addRoute(regexp, transitionStrategy, shellState) {
-    const route = new Route(regexp, transitionStrategy);
+  _addRoute(regexp, transitionStrategy, shellState, onAttached) {
```

Comment:
This definitely needs refactor. But I wanted to discuss the whole WebPack, dynamic loading thing first.